hessian-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,79 @@
+registryItems: {
+  applications: [
+  {
+    hessian-consumer: 3
+  }
+  ],
+  instances: [
+  {
+    hessian-consumer: 1
+  }
+  ],
+  operationNames: [
+  {
+    hessian-consumer: [
+      /HelloService,
+      /hello
+    ]
+  }
+  ]
+},
+"segmentItems: [
+{
+  applicationCode: hessian-consumer,
+  segmentSize: 1,
```

Comment:
Why only one segment? And where is the segment ref?

Reply:
resolving.